doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -519,12 +521,14 @@ When required, the Annex 1 shows how to provision a table for Carto, among other
 [Top](#top)
 
 ####<a name="section2.3.6"></a>Supported Orion's geometries
-Current version of `NGSICartoDBSink` supports the following Orion's geometries:
+Current version of `NGSICartoDBSink` supports the following NGSIv2 geometries:
 
 * `geo:point`, in this case the geolocated attribute is about a single point.
-* `geo:json`, in this case the geolocated attribute is about any geometry, from a simple point to a complex polygon.
+* `geo:json`, despite a GeoJson can describe any geometry, from a simple point to a complex polygon, it must represent a single point.
 * `location` metadata, in this case the geolocated attribute is about a single point.
 
+You can get more information at [NGSIv2]() specification (<i>Geospatial properties of entities</i> section).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1287/commits/ccdf187fadde5a966c09b2a43622c88471c8ecec